tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,742 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true,
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Qiskit Gradient Framework using Primitives\n",
+    "This tutorial demonstrates the use of the `qiskit.algorithms.gradients` module to evaluate quantum gradients using the [Qiskit Primitives](https://qiskit.org/documentation/apidoc/primitives.html).\n",
+    "\n",
+    "## Introduction\n",
+    "The gradient frameworks allows the evaluation of [quantum gradients](https://sci-hub.hkvisa.net/10.1103/PhysRevA.99.032331).\n",
```

Comment:
I think that you can add the 2 references @Zoufalc proposed in the following way (also, as a note, we should not add links to sci-hub, but rather the original source of the paper):

```suggestion
    "The gradient frameworks allows the evaluation of quantum gradients (see [Schuld et al.](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.99.032331) and [Mari et al.](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.103.012405)). \n",
```